### PR TITLE
submitting homework 2

### PR DESCRIPTION
**Pull request by THEJRRR**
Completed Homework 2

### CODE REVIEW

File: week2/homework/questions.txt [hunk 1]
```diff
@@ -3,11 +3,16 @@ Containers, Blocks, and Iterators
 Sharing Functionality: Inheritance, Modules, and Mixins
 
 1. What is the difference between a Hash and an Array?
+A hash stores a whole integer while an array can hold any object.
```
Comment (bethanyr) [inline week2/homework/questions.txt:6]: I think you mixed up hash and array. An array is indexed by an integer. A hash is key value pairs, the key can be any object.

File: week3/homework/calculator_spec.rb [hunk 2]
```diff
@@ -0,0 +1,68 @@
+require "#{File.dirname(__FILE__)}/calculator"
+
+describe Calculator do
```
Comment (bethanyr) [inline week3/homework/calculator_spec.rb:3]: the homework is to write a separate module that will make the tests in the calculator_spec.rb pass.

**Comment by bethanyr** [comment]
I see the following homework submitted: Questions for Week 2, Questions for Week 3. You are missing the coding exercises for week 2 and week 3. You can submit another pull request when you have completed the coding assignments.

**Comment by THEJRRR** [comment]
That is strange,  I don;t know how I would have submitted the questions
without the coding assignments. I may be using git incorrectly. I will try
submitting again tonight.

**Comment by THEJRRR** [comment]
I realized my error and I was using git incorrectly. The coding assignments
should now be added.

Thank you!

**Pull request closed without merging**